boards/arm/ubx_bmd330eval_nrf52810/Kconfig

```diff
@@ -0,0 +1,15 @@
+# BMD-330-EVAL board configuration
+
+# Copyright (c) 2021 u-blox AG
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_ENABLE_DCDC
+	bool "Enable DCDC mode"
+	select SOC_DCDC_NRF52X
+	default y
+	depends on BOARD_UBX_BMD330EVAL_NRF52810
+
+# BT_CTLR depends on BT. When BT is enabled we should default to also
+# enabling the controller.
+config BT_CTLR
+	default y if BT
```

Comment:
This is inconsistent with: https://github.com/zephyrproject-rtos/zephyr/blob/ec796ed3c2b833f0ba6faeb3fb5d467d0e7dcc85/boards/arm/ubx_bmd330eval_nrf52810/Kconfig.defconfig#L11-L15

Reply:
This is an artifact of using nrf52dk_nrf52810 as an example -- removed comments in Kconfig.defconfig